src/libraries/System.Private.CoreLib/src/System/IO/RandomAccess.cs

```diff
@@ -306,14 +297,7 @@ private static void ValidateInput(SafeFileHandle handle, long fileOffset, bool f
                     ThrowHelper.ThrowObjectDisposedException_FileClosed();
                 }
 
-                // NOTE: this function was unconditionally throwing an exception for all non-seekable handles but
```

Comment:
this comment was valuable, please just move it to `FlushToDisk(SafeFileHandle handle)` rather than deleting

Reply:
Done via [this commit](https://github.com/dotnet/runtime/pull/89100/commits/4748b8a41da7235973b159db6adcc023c880fb39) 👍